float32 face_depth
float32 center_depth
float32 face_x
float32 face_y
float32 time
